spec/mailers/user_mailer_spec.rb

```diff
@@ -40,17 +40,23 @@
 
   describe 'reservation_status_update' do
     it 'sends to the reserver' do
+      # make it a request so the email sends
+      @res.update_attributes(flags: Reservation::FLAGS[:request])
```

Comment:
Good call, I didn't see those later tests. That would be fine too, just use the `force` parameter that we've already defined - the point of these first tests is to test the *general* behavior of the mailer, not any specific reservation scenario (although make sure the view won't be trying to pull in any `nil` attributes :stuck_out_tongue:).